src/Weeds3dViewer/Weeds3dViewer.js

```diff
@@ -119,21 +125,114 @@ const Weeds3dViewer = () => {
       type: 'numeric',
       align: 'justify',
       render: (rowData) => {
-        return Math.round((rowData.file_size / 1000000) * 10) / 10 + ' MB';
+        return convertToMB(rowData.file_size) + ' MB';
+      },
+    },
+    {
+      title: 'File Info',
+      align: 'justify',
+      render: (rowData) => {
+        return convertToMB(rowData.file_size) < 25 ? (
+          <Tooltip title="This file size is too small">
+            <Error style={{ color: '#cc4968' }} />
+          </Tooltip>
+        ) : convertToMB(rowData.file_size) > 750 ? (
+          <Tooltip title="This file size is good">
+            <CheckCircle style={{ color: '#2F7C31' }} />
+          </Tooltip>
+        ) : (
+          <Tooltip title="This file size is informational">
+            <Info style={{ color: '#5e7ee0' }} />
+          </Tooltip>
+        );
       },
     },
   ];
 
+  const convertToMB = (size) => {
+    return Math.round((size / 1000000) * 10) / 10;
```

Comment:
The multiply and divide by ten is to get MB rounded to the nearest 0.1. I have no opinion about whether you do MiB or MB (2^10*10^3, or 10^6, or 2^20, depending which definition you're using).

Reply:
I think it's a way to round to 1 decimal place instead of to a whole number 
https://stackoverflow.com/questions/7342957/how-do-you-round-to-1-decimal-place-in-javascript
should I just round to the nearest whole number?

Reply:
eh i don't actually think it matters anymore it just looked weird to me but that's probably just my computer engineer brain acting up. i'll set this as resolved